src/actions/vstudio/vs2010_vcxproj.lua

```diff
@@ -59,66 +58,6 @@
 
 
 --
--- Update the file table with generated files.
---
-	function m.updateFileTable(prj)
-		local function addGeneratedFile(cfg, source, filename)
-			-- mark that we have generated files.
-			cfg.project.hasGeneratedFiles = true
-
-			-- add generated file to the project.
-			local files = cfg.project._.files
-			local node = files[filename]
-			if not node then
-				node = fileconfig.new(filename, cfg.project)
-				files[filename] = node
-				table.insert(files, node)
-			end
-
-			-- always overwrite the vpath and dependency information.
-			if not node.vpath then
-				node.vpath = path.join("Generated", node.name)
-			end
-			node.dependsOn = source
-			node.generated = true
-
-			-- add to config.
-			fileconfig.addconfig(node, cfg)
```

Comment:
I think this line should go inside the `if not node then` : if the file already exists, its file config should also already be there, and adding it again might cause problems.